code/modules/pool/pool_noodles.dm

```diff
@@ -0,0 +1,44 @@
+//Pool noodles
+
+/obj/item/toy/poolnoodle
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "noodle"
+	name = "pool noodle"
+	desc = "A strange, bulky, bendable toy that can annoy people."
+	force = 0
+	color = "#000000"
+	w_class = WEIGHT_CLASS_SMALL
+	throwforce = 1
+	throw_speed = 10 //weeee
+	hitsound = 'sound/weapons/tap.ogg'
+	attack_verb = list("flogged", "poked", "jabbed", "slapped", "annoyed")
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+
+/obj/item/toy/poolnoodle/attack(target as mob, mob/living/user as mob)
+	..()
+	if(prob(80))
+		user.emote("spin")
+	if(prob(5))
+		user.emote("spin")
```

Comment:
are both of these meant to spin the user?

Reply:
I think so?